routes/project/github.rb

```diff
@@ -59,6 +59,9 @@ class Clover
         r.on "runner" do
           r.get true do
             @runners = @project.github_runners_dataset.eager(:vm, :strand).reverse(:created_at).all
+            @runners = @project.github_runners_dataset.eager(:vm, :strand)
+              .exclude(Sequel[:github_runner][:id] => Strand.where(prog: "Vm::GithubRunner", label: ["destroy", "wait_vm_destroy"]).select(:id))
```

Comment:
Another option is to join with a strand table. I'm not sure which one is more efficient, but this version is more readable for me. cc @jeremyevans